### PR TITLE
Adds Codacy Online Linting Checks Configuration

### PR DESCRIPTION
**Pull request by timnolte**
Adds the required configuration needed to Codacy to beter perform checks on the theme code for coding standards.

### CODE REVIEW

File: .codacy.yml [hunk 1]
```diff
@@ -0,0 +1,20 @@
+---
+engines:
+  phpcs:
+    enabled: true
+    php_version: 7.3
+  eslint:
+    enabled: true
+    exclude_paths:
+      - '**/*.svg'
```
Comment (kmonahan) [inline .codacy.yml:9]: Let's also disable remark-lint here, as was added to the Drupal PR: https://github.com/forumone/gesso/pull/441/files#diff-6e27af35533eda2e3e82d6d7f8654c26824396797716d37ac7df096a1b090503R11-R12

**Review by kmonahan — COMMENTED** [review]
@coreylafferty , once .codacy.yml is updated, this is ready to merge.

File: .codacy.yml (hunk 1 above)
Comment (timnolte) [reply .codacy.yml:9]: 👍🏼 thanks I must have missed that change.

Comment (timnolte) [reply .codacy.yml:9]: This has been done and should be good to go now.

**Comment by kmonahan** [comment]
Seeing the same results as https://github.com/forumone/gesso/pull/442#issuecomment-861818057. Running `f1 up` or `f1 run gesso gulp build` while within the theme directory runs without errors but doesn't start the site/display the assets on the front-end. Running the same commands one directory higher works as expected on my local.

**Comment by coreylafferty** [comment]
Closing in lieu of #254

**Pull request closed without merging**